docs/proposals/kuratorcd/kurator-cd.md

```diff
@@ -0,0 +1,622 @@
+---
+
+## Kurator's Continuous Delivery design
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+### Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. 
+
+A good summary is probably at least a paragraph in length.
+-->
+
+Kurator, as an open-source distributed cloud-native platform, has been pivotal in aiding users to construct their distributed cloud-native infrastructure, thereby facilitating enterprise digital transformation.
+
+To further enhance its functionality, this proposal designs Kurator's Continuous Delivery feature to meet users' needs for automated distribution of verified code.
+
+By integrating Flagger, we aim to provide users with reliable, fast and unified Continuous Delivery, enabling them to easily distribute code across multiple clusters.
+
+Base on Flagger, Kurator also offers A/B, Blue/Green and Canary distribution options. Meet the needs of the test verification release.
+
+### Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users.
+-->
+
+With the increase in project size and complexity and the development of cloud computing technology, the CI/CD process has been proposed.
+
+CI/CD has many advantages such as increased development efficiency, improved quality, more reliable deployment and better continuous learning and improvement, which is more suitable for today's software development process.
+
+Kurator is an open source distributed cloud native suite that provides users with a one-stop open source solution for distributed cloud native scenarios.
+
+Therefore, CI/CD as an important feature of cloud native usage scenarios, Kurator needs to provide relevant functional support to achieve the vision of Kurator unified configuration distribution.
+
+#### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+Unified configuration distribution only requires the user to declare the desired API state in one place, and Kurator will automatically handle all subsequent operations.
+
+Kurator deploys different daemons on different nodes in the cluster with different rules for configuration distribution based on the declarations in the Spec.
+
+In Kurator, you can choose to distribute the application with the same configuration to a specific single or multiple clusters for verification.
+
+- **unified Continuous Delivery**
+    - Supports unified configuration of releases for multiple clusters. Achieve the deployment configuration of the application to be distributed to the specified single or multiple clusters.
+    - Support A/B, Blue/Green, Canary three release validation rules configuration.
```

Comment:
These three are all deploy strategies, donot mention validation? It is confusing to users